saleor/graphql/product/sorters.py

```diff
@@ -131,7 +131,7 @@ class ProductOrderField(graphene.Enum):
     TYPE = ["product_type__name", "name", "slug"]
     PUBLISHED = ["is_published", "name", "slug"]
     PUBLICATION_DATE = ["publication_date", "name", "slug"]
-    COLLECTION = ["sort_order"]
+    COLLECTION = ["collectionproduct__sort_order"]
```

Comment:
We should add a unique field as the last element of this list. Without it, pagination doesn't work correctly.